pkgs/net-doc/net/scribblings/osx-ssl.scrbl

```diff
@@ -3,11 +3,11 @@
           (for-label net/osx-ssl
                      openssl))
 
-@title[#:tag "osx-ssl"]{OS X Native SSL: Secure Communication}
+@title[#:tag "osx-ssl"]{macOS Native SSL: Secure Communication}
```

Comment:
I’ll revert this one because “OS X” is baked into the whole naming scheme of the package & its functions. The package maintainer should decide what happens.